packages/bitlib/bitlib.0.1.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+synopsis: "A library for writing binary files"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/bitlib"
+bug-reports: "https://github.com/chrisnevers/bitlib/issues"
+depends: ["ocaml" "ocamlbuild"]
+build: make
```

Comment:
Looking at the code, it seems to me that you are only building the library and don’t install anyhing

Reply:
Thanks for spotting that, I am a complete beginner and would appreciate any help in getting this working/prepared. What do I have to do?

Reply:
I think the easiest way to make sure that you buil and install the library is to use dune as build system